sessionctx/variable/tidb_vars.go

```diff
@@ -520,6 +520,9 @@ const (
 
 	// TiDBTrackAggregateMemoryUsage indicates whether track the memory usage of aggregate function.
 	TiDBTrackAggregateMemoryUsage = "tidb_track_aggregate_memory_usage"
+
+	// TiDBEnableTiFlashFallbackTiKV indicates whether to fallback to TiKV when TiFlash is unavailable.
+	TiDBEnableTiFlashFallbackTiKV = "tidb_enable_tiflash_fallback_tikv"
```

Comment:
Please add an integration test for this variable. You can add it in `executor/set_test.go:TestSetVar`.